### ISSUE TITLE
Missing the backend module on the sdist distribution

### ISSUE THREAD

**Issue by david-caro** [sh4nks/flask-caching#15]
It seem that the current release is missing the backend internal module, that makes the code break when installing it from pip (when using the source is ok).

For example:
```
[2016-12-09 11:05:41,383] ERROR in app: Failed to initialize entry point: inspire_cache = inspirehep.modules.cache.ext:INSPIRECache
Traceback (most recent call last):
  File "/virtualenv/bin/inspirehep", line 11, in <module>
    load_entry_point('Inspirehep', 'console_scripts', 'inspirehep')()
  File "/virtualenv/lib/python2.7/site-packages/click/core.py", line 716, in __call__
    return self.main(*args, **kwargs)
  File "/virtualenv/lib/python2.7/site-packages/flask/cli.py", line 345, in main
    return AppGroup.main(self, *args, **kwargs)
  File "/virtualenv/lib/python2.7/site-packages/click/core.py", line 696, in main
    rv = self.invoke(ctx)
  File "/virtualenv/lib/python2.7/site-packages/click/core.py", line 1060, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/virtualenv/lib/python2.7/site-packages/click/core.py", line 889, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/virtualenv/lib/python2.7/site-packages/click/core.py", line 534, in invoke
    return callback(*args, **kwargs)
  File "/virtualenv/lib/python2.7/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/virtualenv/lib/python2.7/site-packages/flask/cli.py", line 228, in decorator
    with __ctx.ensure_object(ScriptInfo).load_app().app_context():
  File "/virtualenv/lib/python2.7/site-packages/flask/cli.py", line 201, in load_app
    rv = self.create_app(self)
  File "/virtualenv/lib/python2.7/site-packages/invenio_base/app.py", line 159, in create_cli_app
    app = create_app(debug=get_debug_flag())
  File "/virtualenv/lib/python2.7/site-packages/invenio_base/app.py", line 120, in _create_app
    modules=extensions,
  File "/virtualenv/lib/python2.7/site-packages/invenio_base/app.py", line 179, in app_loader
    modules=modules)
  File "/virtualenv/lib/python2.7/site-packages/invenio_base/app.py", line 230, in _loader
    init_func(ep.load())
  File "/virtualenv/lib/python2.7/site-packages/invenio_base/app.py", line 178, in <lambda>
    _loader(app, lambda ext: ext(app), entry_points=entry_points,
  File "/code/inspirehep/modules/cache/ext.py", line 37, in __init__
    self.init_app(app)
  File "/code/inspirehep/modules/cache/ext.py", line 41, in init_app
    self.cache = Cache(app)
  File "/virtualenv/lib/python2.7/site-packages/flask_caching/__init__.py", line 150, in __init__
    self.init_app(app, config)
  File "/virtualenv/lib/python2.7/site-packages/flask_caching/__init__.py", line 191, in init_app
    self._set_cache(app, config)
  File "/virtualenv/lib/python2.7/site-packages/flask_caching/__init__.py", line 196, in _set_cache
    from . import backends
ImportError: cannot import name backends
```

Cheers!

**Comment by david-caro** [comment]
The list of installed packages in the virtualenv:

```
12:11 PM ~/.docker_data/tmp/virtualenv/lib/python2.7/site-packages/flask_caching 
dcaro@pcrcssis001$ pwd
/home/dcaro/.docker_data/tmp/virtualenv/lib/python2.7/site-packages/flask_caching
12:14 PM ~/.docker_data/tmp/virtualenv/lib/python2.7/site-packages/flask_caching 
dcaro@pcrcssis001$ tree
.
├── _compat.py
├── _compat.pyc
├── __init__.py
├── __init__.pyc
├── jinja2ext.py
└── jinja2ext.pyc

0 directories, 6 files
```

**Comment by sh4nks** [comment]
Should be fixed with 1.1.1. Could you try again?

**Comment by david-caro** [comment]
It is 👍, thanks!!
